pkg/virt-handler/node-labeller/cpu_plugin.go

```diff
@@ -82,15 +81,15 @@ func (n *NodeLabeller) loadDomCapabilities() error {
 	usableModels := make([]string, 0)
 	for _, mode := range hostDomCapabilities.CPU.Mode {
 		if mode.Name == v1.CPUModeHostModel {
-			if virtconfig.IsARM64(n.arch) {
+			if !n.Arch.supportsHostModel() {
 				log.Log.Warning("host-model cpu mode is not supported for ARM architecture")
 				continue
 			}
 
 			n.cpuModelVendor = mode.Vendor.Name
 			// On s390x the xml does not include a CPU Vendor, however there is only one company selling them anyway.
```

Comment:
Comment does not relate to code below anymore?

Reply:
Yes, i'll move the comment into the s390x.go file instead.